### PR TITLE
Move progress bar in ScheduleView

### PR DESCRIPTION
**Pull request by rock3r**
Progress was broken, we did progress on this. We even fixed time 🍌

### CODE REVIEW

File: app/src/main/java/com/connfa/schedule/domain/view/SchedulePage.java [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
 @AutoValue
 public abstract class SchedulePage {
 
-    // todo parse date, format it
+    private static final DateTimeFormatter FORMATTER = DateTimeFormat.forPattern("d-MM-yyyy");
+
     public static SchedulePage create(String date, List<Event> events) {
-        return new AutoValue_SchedulePage(date, events);
+        DateTime dateTime = DateTime.parse(date, FORMATTER);
+        String title = dateTime.toString("EEE d");
```
Comment (frapontillo) [inline app/src/main/java/com/connfa/schedule/domain/view/SchedulePage.java:18]: WHAT IS THIS EXTRACT PLEASE

File: app/src/main/res/layout/activity_schedule.xml [hunk 2]
```diff
@@ -35,4 +35,10 @@
     android:layout_height="match_parent"
     app:layout_behavior="@string/appbar_scrolling_view_behavior" />
 
-</android.support.design.widget.CoordinatorLayout>
+  <ProgressBar
```
Comment (frapontillo) [inline app/src/main/res/layout/activity_schedule.xml:38]: 👍  I like this hihihihihihi

**Review by frapontillo — CHANGES_REQUESTED** [review]
❌

File: app/src/main/res/layout/activity_schedule.xml (hunk 2 above)
Comment (rock3r) [reply app/src/main/res/layout/activity_schedule.xml:38]: Dio cane

File: app/src/main/java/com/connfa/schedule/domain/view/SchedulePage.java (hunk 1 above)
Comment (rock3r) [reply app/src/main/java/com/connfa/schedule/domain/view/SchedulePage.java:18]: Next PIERRE

**Review by frapontillo — APPROVED** [review]

File: app/src/main/res/layout/activity_schedule.xml (hunk 2 above)
Comment (danybony) [reply app/src/main/res/layout/activity_schedule.xml:38]: porcatroiadavvero

Comment (frapontillo) [reply app/src/main/res/layout/activity_schedule.xml:38]: 💯

**Pull request merged**